src/phoenix/server/api/types/DatasetExample.py

```diff
@@ -78,8 +79,17 @@ async def experiment_runs(
             .options(joinedload(models.ExperimentRun.trace).load_only(models.Trace.trace_id))
             .join(models.Experiment, models.Experiment.id == models.ExperimentRun.experiment_id)
             .where(models.ExperimentRun.dataset_example_id == example_id)
-            .order_by(models.Experiment.id.desc())
         )
+        if experiment_ids is not UNSET:
```

Comment:
```suggestion
        if experiment_ids:
```

Just looking at the type, `experiment_ids` could also be `None`.